scripts/generate_icon_testdata.sh

```diff
@@ -12,19 +12,10 @@ if [[ -z "$PROBTEST_DATA" ]]; then
     exit 1
 fi
 
-for exp in atm_amip_test mch_opr_r04b07
+for exp in mch_opr_r04b07
 do
-    if [[ "$exp" == "atm_amip_test" ]]; then
-        file_id=(
-            --file-id NetCDF "*atm_3d*.nc"
-            --file-id NetCDF "*lnd*.nc"
-        )
-    fi
-    if [[ "$exp" == "mch_opr_r04b07" ]]; then
-        file_id=(
-            --file-id NetCDF "*atm_3d_ml*.nc"
-        )
-    fi
+    file_id=(
+        --file-id NetCDF "*atm_3d_ml*.nc"
```

Comment:
Would it make sense to still have a test case with more than one file ID?

Reply:
In principal yes. But I would like to make the testing independent of daint and of ICON, so I would prefer to do first that and then increase the test coverage. What do you think?

Reply:
Yes, we can do it like this.

Reply:
```suggestion
    file_id=(
        --file-id NetCDF "*atm_3d_ml*.nc"
    )
```

Reply:
Still this needs to be fixed ;-)